src/middlewares/customValidator.js

```diff
@@ -0,0 +1,54 @@
+import Response from '../helpers/Response';
+
+
+/**
+ * @exports
+ * @class UserController
+ */
+class customValidator {
+  /**
+   * User can be able to sign up
+   * @static
+   * @param {object} req  request object
+   * @param {object} res response object
+   * @param {object} next next
+   * @memberof UserController
+   * @returns {object} data
+   */
+  static isCoordinates(req, res, next) {
+    const googleCoordinatesPattern = /^(-?\d+(\.\d+)?),\s*(-?\d+(\.\d+)?)$/;
+    if (req.body.googleCoordinates.match(googleCoordinatesPattern)) {
+      return next();
+    }
+    return Response.errorMessage(req, res, 'invalid google maaps coordinates', 400);
+  }
+
+  /**
+   * User can be able to sign up
+   * @static
+   * @param {object} req  request object
+   * @param {object} res response object
+   * @param {object} next next
+   * @memberof UserController
+   * @returns {object} data
+   */
+  static isImage(req, res, next) {
+    const imageURL = req.body.imageURL ? [req.body.imageURL] : req.body.imageUrls;
+    let isImage;
+    imageURL.forEach((imageUrl) => {
+      const extension = imageUrl.substring(
+        imageUrl.lastIndexOf('.') + 1
+      ).toLowerCase();
+      if (extension === 'gif' || extension === 'png' || extension === 'bmp'
+      || extension === 'jpeg' || extension === 'jpg') {
+        isImage = true;
```

Comment:
I think we can improve the code by using ES6 includes methods. For example
- Create an array of extensions.
- Check if extensions exists in array.
- Includes methods return `true` or `false`.

``` js
    const arrayOfExtension = ['gif', 'png', 'bmp', 'jpeg', 'jpg']; 
    if ( arrayOfExtension.includes(comingExtensions)){
          do something
    }
```
afterward, we check if extensions includes

Reply:
Wow! thanks alot..